homeassistant/components/person/__init__.py

```diff
@@ -428,18 +428,22 @@ def ws_list_person(hass: HomeAssistantType,
                            connection: websocket_api.ActiveConnection, msg):
     """Create a person."""
     manager = hass.data[DOMAIN]  # type: PersonManager
-    person = await manager.async_create_person(
-        name=msg['name'],
-        user_id=msg.get('user_id'),
-        device_trackers=msg['device_trackers']
-    )
-    connection.send_result(msg['id'], person)
+    try:
+        person = await manager.async_create_person(
+            name=msg['name'],
+            user_id=msg.get('user_id'),
+            device_trackers=msg['device_trackers']
+        )
+        connection.send_result(msg['id'], person)
+    except ValueError as err:
```

Comment:
When does it happen?

Reply:
duplicate user id

Reply:
Right. Delete person also raises `ValueError`.